tuplex/adapters/cpython/include/PythonHelpers.h

```diff
@@ -267,7 +267,7 @@ namespace python {
      * @param type specify what type of objects should be serialized, may contain options.
      * @return Tuplex C++ row object.
      */
-    extern tuplex::Row pythonToRow(PyObject *obj, const python::Type &type);
+    extern tuplex::Row pythonToRow(PyObject *obj, const python::Type &type, bool autoUpcast);
```

Comment:
add description to doxygen comment what autoUpcast does/means. Maybe set default value? E.g., false?